content/en/docs/contribute/_index.md

```diff
@@ -29,35 +29,35 @@ about contributing to Kubernetes.
 
 This website is maintained by [Kubernetes SIG Docs](/docs/contribute/#get-involved-with-sig-docs).
 
-Kubernetes documentation contributors:
+Kubernetes docs contributors can:
 
 - Improve existing content
 - Create new content
-- Translate the documentation
-- Manage and publish the documentation parts of the Kubernetes release cycle
+- Translate the docs
+- Manage and publish the docs parts of the Kubernetes release cycle
 
 
 
 <!-- body -->
 
 ## Getting started
 
-Anyone can open an issue about documentation, or contribute a change with a
+Anyone can open an issue about docs, or contribute a change with a
 pull request (PR) to the
 [`kubernetes/website` GitHub repository](https://github.com/kubernetes/website).
-You need to be comfortable with
-[git](https://git-scm.com/) and
-[GitHub](https://lab.github.com/)
-to work effectively in the Kubernetes community.
+
+Firstly, you need:
+
+- Be familiar with [git](https://git-scm.com/).
+- Have account in [GitHub](https://github.com/).
 
 To get involved with documentation:
 
-1. Sign the CNCF [Contributor License Agreement](https://github.com/kubernetes/community/blob/master/CLA.md).
-2. Familiarize yourself with the [documentation repository](https://github.com/kubernetes/website)
-   and the website's [static site generator](https://gohugo.io).
-3. Make sure you understand the basic processes for
-   [opening a pull request](/docs/contribute/new-content/open-a-pr/) and
-   [reviewing changes](/docs/contribute/review/reviewing-prs/).
+1. Sign the [CNCF Contributor License Agreement](https://github.com/kubernetes/community/blob/master/CLA.md).
```

Comment:
This is an ordered list & should follow the style guide. Please make the necessary amendments.

https://kubernetes.io/docs/contribute/style/style-guide/